src/pocketmine/network/mcpe/protocol/types/PlayerListEntry.php

```diff
@@ -40,6 +40,12 @@ class PlayerListEntry{
 	public $xboxUserId;
 	/** @var string */
 	public $platformChatId = "";
+	/** @var int */
+	public $buildPlatform = 0; //Unknown
```

Comment:
Could you rename this to `deviceOs`? I think this is a more suitable name for this field.
And it's better to use -1 as the default value: -1 = UNKNOWN_OS